src/App.js

```diff
@@ -92,11 +92,16 @@ function App(props) {
               gas: "300000000000000",
               bundle: false,
             }),
+            setupFastAuth({
+              networkId: NetworkId,
+              signInContractId: "v1.social08.testnet",
```

Comment:
@gutsyphilip Not blocking for us getting this in for testnet testing, but we can probably grab this based on `networkId` from https://github.com/NearSocial/VM/blob/10064520f3b752cffcbeb13133b5c97f5d287e4a/src/lib/data/near.js#L22